Tests/Categories/NSString_DPLQuerySpec.m

```diff
@@ -73,7 +73,16 @@
     it(@"should serialize multiple arrays from dictionary into the query string", ^{
         NSDictionary *params = @{ @"beers": @[ @"stout", @"ale" ], @"liquors": @[ @"vodka", @"whiskey" ] };
         NSString *query = [NSString DPL_queryStringWithParameters:params];
-        expect(query).to.equal(@"beers[]=stout&beers[]=ale&liquors[]=vodka&liquors[]=whiskey");
+
+        NSString *queryStringToMatch;
+        NSString *queryStringPartBeers = @"beers[]=stout&beers[]=ale";
+        NSString *queryStringPartLiquors = @"liquors[]=vodka&liquors[]=whiskey";
+        if ([[params allKeys][0] isEqualToString:@"beers"]) {
```

Comment:
I think we should probably have a suitable default here such as ordered ascending that way we can expect a consistent result. 

Similarly, when parsing a url, I would expect to have the parameter order preserved so re-serialization mirrors the original url though that can't really happen in the category (#15). 

For example:
```objc
NSURL *url = [NSURL URLWithString:@"dpl://something/?liquors[]=vodka&liquors[]=whiskey&beers[]=stout&beers[]=ale"];
DPLDeepLink *link = [[DPLDeepLink alloc] initWithURL:url];
link.URL.query  // should serialize as => @"liquors[]=vodka&liquors[]=whiskey&beers[]=stout&beers[]=ale"
```